src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -40,6 +40,7 @@
 import jakarta.inject.Named;
 import jakarta.persistence.*;
 import jakarta.persistence.criteria.*;
+import org.jsoup.internal.StringUtil;
```

Comment:
I think we usually use org.apache.commons.lang3.StringUtils for isBlank() (org.jsoup.internal.StringUtil isn't used anywhere currently)